internal/pkg/cli/cli.go

```diff
@@ -136,25 +136,26 @@ func quoteStringSlice(in []string) []string {
 	return quoted
 }
 
-// CwdPathDisplayer displays paths to the user relative to their current working directory.
-type CwdPathDisplayer struct {
-	workingDir string
-}
-
-func NewCwdPathDisplayer() (*CwdPathDisplayer, error) {
-	workingDir, err := os.Getwd()
+// displayPath takes any path and returns it in a form ready to be displayed to
+// the user on the command line.
+//
+// No guarantees are given on the stability of the path across runs, all that is
+// guaranteed is that the displayed path is visually pleasing & meaningful for a
+// user.
+//
+// This path should not be stored in configuration files or used in any way except
+// for being displayed to the user.
+func displayPath(target string) string {
+	base, err := os.Getwd()
+
+	if err != nil || !filepath.IsAbs(target) {
```

Comment:
nit: we can move the `os.Getwd` call after the if-block since `base` isn't used in the if-statement